### PR TITLE
Viz tuneup

### PR DESCRIPTION
**Pull request by paulstgermain**
## Description

This update:

- Fixes broken judge page links on case and judge tables
- Removes extra mock visualizations, fine-tunes the styling on the relevant visualizations
- Adds matching headers to all components

Fixes # (issue)
Table links for judges now work.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by sebastian-espeset — APPROVED** [review]
Thanks paul, can't wait to see them

File: src/components/pages/Cases/CaseTable.js [hunk 1]
```diff
@@ -193,7 +194,9 @@ export default function CaseTable(props) {
       sorter: (a, b) => a.last_name.localeCompare(b.last_name),
       sortDirections: ['descend', 'ascend'],
       ...getColumnSearchProps('judge_name'),
-      render: text => <Link to={`/judge/${casesData.judge_id}`}>{text}</Link>,
+      render: (text, record) => (
+        <Link to={`/judge/${record.judge_id}`}>{text}</Link>
+      ),
```
Comment (dgamboa) [inline src/components/pages/Cases/CaseTable.js:199]: Good addition of record as a its own argument here

File: src/components/pages/DataHub/DataHub.js [hunk 2]
```diff
@@ -286,15 +203,17 @@ const DataHub = props => {
...
+      </p>
       <div className="mainChartContainer">
         <TestDataChart />
       </div>
-      <h2>Ratio of Cases Resulting In Granted Asylum: {grantedRatio}%</h2>
 
       <div className="subChartsContainer">
         <CaseDataChart />
-        <CaseDataChart />
+        <h2>Ratio of Cases Resulting In Granted Asylum: {grantedRatio}%</h2>
```
Comment (dgamboa) [inline src/components/pages/DataHub/DataHub.js:216]: Great title

File: src/components/pages/JudgePage/JudgePage.js [hunk 3]
```diff
@@ -87,12 +90,14 @@ export default function JudgePage(props) {
...
         }}
-        style={{ width: '40%', height: '25rem' }}
+        style={{ width: '100%', height: '25rem' }}
       />
     );
   };
 
   const ProGroundsChart = () => {
+    vizData.bar_protected_grounds.layout.title =
+      'Outcomes by Protected Grounds';
```
Comment (dgamboa) [inline src/components/pages/JudgePage/JudgePage.js:100]: Another great title

**Review by dgamboa — APPROVED** [review]
Looks great -- these are a solid step fwd in displaying visualizations

**Pull request merged**